src/egon/data/datasets/etrago_setup.py

```diff
@@ -247,16 +253,18 @@ class EgonPfHvStorage(Base):
     p_set = Column(Float(53))
     q_set = Column(Float(53))
     sign = Column(Float(53), server_default="1")
-    marginal_cost = Column(Float(53))
-    capital_cost = Column(Float(53))
-    state_of_charge_initial = Column(Float(53))
-    cyclic_state_of_charge = Column(Boolean)
+    marginal_cost = Column(Float(53), server_default="0.")
```

Comment:
I doubt that I like these server_default values. I got rid of them on purpose so that we actually see where we still do not have data. With these default values it might seem that we have nicely filled data if in case we don't... Especially s.th. like lifetime set to 'inf' could get very weird annualization results. Do you wanna get rid of NA values which are annoying (but maybe it's good that they annoy us ;)) or what is the idea behind this?

Reply:
We added these pypsa-default values to see all the parameters for eTraGo/PyPSA in the database. The infinite lifetime is maybe not the best example for this, but e.g. in case of `p_nom_min` or `marginal_cost` I liked the idea to directly see which data is used.

Reply:
Ah ok, I think I understand. I guess both ways have its pros and cons. I am still not very much convinced about these server_defaults. Apparently it is s.th. only used as sort of a development or debugging functionality. Maybe we can agree on that the goal for the final data set should be that no server defaults are still defining anything and that instead all parameters are explicitly defined. So may be we can add an issue to get rid of these defaults once we feel our data sets are complete? This can be seen as a little plausibility check, too... What do you think?

Reply:
I think I would like to move this discussion into our weekly meeting...